### ISSUE TITLE
IP/NET with trailing space error in 'access.conf'

### ISSUE THREAD

**Issue by serge-vk** [mrash/fwknop#288]
I've just discovered that when I use an address formatted with a net mask (192.168.1.0/24) in access.conf, if there is a trailing space in the SOURCE or DESTINATION statements, then fwknopd bails out with an error like this:
```
[*] Invalid IP mask str '24 '.
[*] Fatal invalid DESTINATION in access stanza
```
When there is no mask given in the address, the error is not triggered with one to three trailing  chars (the first has to be a space, the 2nd & 3rd can be either space or `#`), but 4 will generate an error. There may be some other interesting combinations, but I think it is pretty obvious that white-space and comment parsing does not work as intended. Unfortunately, an error caused by a trailing space may be quite difficult to locate, because it is not visible in the .conf file. It took me a while to find this out.

**Comment by mrash** [comment]
Thanks - agreed that behavior is inconsistent. Will take a look.

**Comment by mrash** [comment]
Fixed in bacd054.

**Comment by serge-vk** [comment]
Thanks, I will give it a spin :)
